apis/experimentapi/experimentapi.yaml

```diff
@@ -0,0 +1,661 @@
+                    value:
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-23 06:42:39:001+0000'
+                      params:
+                        resmsgid: 8c244093-e19d-4694-a4ba-342bc0f8b721
+                        status: failed
+                        err: CLIENT_ERROR
+                        errorMsg:
+                          status: failed
+                          data.endDate: End_Date should be greater than today's date.
+                      responseCode: CLIENT_ERROR
+                      result: null
+                  StartEndDateValidation:
+                    summary: Start_End_Date_Validation
+                    description: Response example with all required fields
+                    value:
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-23 06:42:39:001+0000'
+                      params:
+                        resmsgid: 8c244093-e19d-4694-a4ba-342bc0f8b721
+                        status: failed
+                        err: CLIENT_ERROR
+                        errorMsg:
+                          status: failed
+                          data.startDate: Date range should not be -ve. Please check your start_date & end_date
+                      responseCode: CLIENT_ERROR
+                      result: null
+          '500':
+            description: Internal Server Error example for experiment create
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/Experiment-500InternalServerError'
+                example:
+                  id: ekstep.analytics.experiment.create
+                  ver: v1
+                  ts: '2020-11-23 15:15:18:331+0000'
+                  params:
+                    resmsgid: null
+                    msgid: null
+                    err: INTERNAL_ERROR
+                    status: SERVER_ERROR
+                    errmsg: 'Process failed,please try again later.'
+                  responseCode: SERVER_ERROR
+                  result: {}
+        deprecated: false
+        security: []
+    /data/v1/experiement/get/'{experiment-id}':
+      get:
+        tags:
+          - V1
+        summary: GetExperiment
+        description:  |-
+          This API is used for get particular experiment based on experiment id
+             - The endpoint for ** Get Experiment** is `/{experiment-id}`
+             - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+        operationId: GetExperiment
+        parameters:
+          - name: experiment-id
+            in: path
+            description: ''
+            required: true
+            style: simple
+            schema:
+              type: string
+        responses:
+          '200':
+            description: Success example for get particular experiment
```

Comment:
An example for a successful retrieval of an existing experiment using the Get Experiment API.